--- src\main\java\example\ArrayDemo.java
+++ src\main\java\example\ArrayDemo.java
@@ -25,11 +25,11 @@
         this.permissionsToGive = Arrays.copyOf(permissionsToGive, permissionsToGive.length);
         return this;
-    }
-
-    public ArrayDemo withPermissionsToNeeded(String[] permissionsToNeeded) {
-        this.permissionsToNeeded = permissionsToNeeded;
-        return this;
-    }
-
+    }
+
+    public ArrayDemo withPermissionsToNeeded(String[] permissionsToNeeded) {
+        this.permissionsToNeeded = permissionsToNeeded.clone();
+        return this;
+    }
+
     public ArrayDemo withPermissionsToNotNeeded(String[] permissionsToNotNeeded) {
         this.permissionsToNotNeeded = permissionsToNotNeeded;
